caddyhttp/browse/browse.go

```diff
@@ -504,7 +504,9 @@ func (b Browse) ServeListing(w http.ResponseWriter, r *http.Request, requestedFi
 
 	}
 
-	buf.WriteTo(w)
+	if _, err = buf.WriteTo(w); err != nil {
+		return http.StatusInternalServerError, err
```

Comment:
We've already written to the response, so instead we should just ignore this error.

We should be able to satisfy errcheck by doing `_, _ = buf.WriteTo(w)`... I think.

Reply:
Oh yeah, sorry about that.